### DISCUSSION TITLE
Edit mode not rendering editables

### DISCUSSION THREAD

**Discussion by sicnarf06** [pimcore/pimcore#15433] [Q&A / Support]
I am using PIMCore 10.6.1 version.
I was able to create a new document but I cannot see the Editables I have setup.

Upon checking the browser console it says **Uncaught ReferenceError: pimcore is not defined**, which is from the startup.js from pimcoreadmin bundle.

Following the script, pimcore variable should have been defined in the common.js from pimcoreadmin bundle as well. But somehow it is not being rendered by my current setup.

Since the scripts are from the pimcore bundle, how can I resolved this issue?

**Comment by robertSt7** [comment]
You can try setting `unminified_js=true` and then see where exactly the error is thrown.
https://pimcore.com/docs/platform/Pimcore/Development_Tools_and_Details/Magic_Parameters#unminified_js
